### PR TITLE
update readme links

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +81,7 @@ that should be
 
 ## Run with Docker
 
-Please refer to [the Docker README](./docker/README.md) for instructions to run `ziti-edge-tunnel` with Docker.
+Please refer to [the container README](https://docs.openziti.io/docs/reference/tunnelers/linux/container/) for instructions to run `ziti-edge-tunnel` with Docker.
```
Comment (dovholuknf) [inline README.md:84]: can we come up with a better way than this that works for both the doc site and someone that checks the repo out?

Comment (qrkourier) [reply README.md:84]: Currently, the ./docker/README.md file in this rep (tunneler SDK) is copied into the Docusaurus build at the link target in this PR.

**Comment by qrkourier** [comment]
@dovholuknf ready for review

**Comment by qrkourier** [comment]
@dovholuknf  ready for review

**Comment by qrkourier** [comment]
@dovholuknf ready for review

File: README.md [hunk 2]
```diff
@@ -9,9 +9,8 @@ that are useful to Ziti Tunnelers.
 
 ## What's a Ziti Tunneler?
 
-The main article about tunnellers is
-https://openziti.github.io/ziti/clients/tunneler.html. Editors may follow the
-"Improve this doc" link on any page.
+[The main article about tunnellers is here](https://docs.openziti.io/docs/reference/tunnelers/linux/). Editors may follow the
```
Comment (dovholuknf) [inline README.md:12]: > Editors may follow the "Edit this page" link on every page.

probably not entirely entirely accurate. that's still a wip on the docs stuff but... /shrug

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**